src/Discord.Net.Rest/Entities/Messages/RestUserMessage.cs

```diff
@@ -148,6 +148,17 @@ public Task ModifySuppressionAsync(bool suppressEmbeds, RequestOptions options =
             TagHandling roleHandling = TagHandling.Name, TagHandling everyoneHandling = TagHandling.Ignore, TagHandling emojiHandling = TagHandling.Name)
             => MentionUtils.Resolve(this, 0, userHandling, channelHandling, roleHandling, everyoneHandling, emojiHandling);
 
+        /// <inheritdoc />
+        public async Task CrosspostAsync(RequestOptions options = null)
+        {
+            if (!(Channel is RestNewsChannel))
+            {
+                throw new InvalidOperationException("Publishing (crossposting) is only valid in news channels.");
```

Comment:
Please document the exception with `<exception>` tag. See [Sandcastle <exception> tag documentation](https://ewsoftware.github.io/XMLCommentsGuide/html/bbd1e65d-c87c-4b46-9a1a-259d3c5cd936.htm).

Reply:
Resolved. And noted for next time.